lib/OpenQA/BuildResults.pm

```diff
@@ -61,6 +61,13 @@ sub count_job {
     return;
 }
 
+sub add_review_badge {
+    my ($build_res) = @_;
+
+    $build_res->{reviewed_all_passed} = $build_res->{passed} == $build_res->{total};
+    $build_res->{reviewed} = $build_res->{failed} > 0 && $build_res->{labeled} == $build_res->{failed};
```

Comment:
shouldn't we change this now to say `$build_res->{labeled} >= $build_res->{failed}` in case we label any other jobs additionally?